docs/using-gitbase/indexes.md

```diff
@@ -0,0 +1,16 @@
+# Indexes
+
+`gitbase` allows you to speed up queries creating indexes.
+
+Indexes are implemented as bitmaps using [pilosa](https://github.com/pilosa/pilosa) as a backend storage for them. To run a pilosa instance see the [getting started](./getting-started.md) section.
+
+Thus, to create indexes you must specify pilosa as the type of index.
+```
+CREATE INDEX files_path_idx ON files USING pilosa (file_path)
+```
+
+Note that you can create an index either **on one or more columns** or **on a single expression**.
+
+You can find some more examples in the [examples](./examples.md) section.
+
+See [go-mysql-server](https://github.com/src-d/go-mysql-server) documentation for more details
```

Comment:
We should link to the specific version, as it's done in some other parts of the docs

Reply:
Done, note that for the moment this link doesn't take you to the `Indexes` section in the `README` because there's no such a section in this revision. When https://github.com/src-d/go-mysql-server/pull/264 will be merged we will update the`go-mysql-server` dependency and get the correct link.